hw/top_earlgrey/dv/chip_smoketests.hjson

```diff
@@ -70,17 +70,23 @@
       en_run_modes: ["sw_test_mode_test_rom"]
     }
     {
-      name: chip_sw_rv_plic_smoketest
+      name: chip_sw_pwrmgr_smoketest
       uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["//sw/device/tests:rv_plic_smoketest:1"]
+      sw_images: ["//sw/device/tests:pwrmgr_smoketest:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=10000000"]
     }
     {
-      name: chip_sw_pwrmgr_smoketest
+      name: chip_sw_pwrmgr_usbdev_smoketest
       uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["//sw/device/tests:pwrmgr_smoketest:1"]
+      sw_images: ["//sw/device/tests/sim_dv:pwrmgr_usbdev_smoketest:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+    }
+    {
+      name: chip_sw_rv_plic_smoketest
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:rv_plic_smoketest:1"]
```

Comment:
no change is made to `chip_sw_rv_plic_smoketest`, except moving it to after chip_sw_p*, in order to make the tests in alphabet order